vendor/wp-now/src/wp-now.ts

```diff
@@ -100,17 +100,18 @@ export default async function startWPNow(
 
 	await prepareWordPress( php, options );
 
+	await installationSteps( php, options );
+
 	if ( options.blueprintObject ) {
 		output?.log( `blueprint steps: ${ options.blueprintObject.steps.length }` );
-		const compiled = compileBlueprint( options.blueprintObject, {
+		const compiled = await compileBlueprint( options.blueprintObject, {
 			onStepCompleted: ( result, step: StepDefinition ) => {
 				output?.log( `Blueprint step completed: ${ step.step }` );
 			},
 		} );
 		await runBlueprintSteps( compiled, php );
 	}
 
-	await installationSteps( php, options );
```

Comment:
In the current implementation, it breaks site creation when blueprint is used as it tries to activate plugins before installing WordPress. The change won't affect behavior for Studio sites, as `options.blueprintObject ` will remain empty for all sites for now, and the issue only pops out when we modify code to provide blueprint.